cmd/chronicle/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gookit/color"
+
+	"github.com/anchore/chronicle/cmd/chronicle/cli"
+)
+
+func main() {
+	root := cli.New()
+
+	if err := root.Execute(); err != nil {
```

Comment:
we should still port the explicit signal handling so that we are cancelling via the context first, and on another signal, we exit 1. Though ctrl C without these explicit signal handlers in place works, the execution path is very different.